frontend/src/pages/RegisterPage.vue

```diff
@@ -0,0 +1,230 @@
+        this.passwordConfirm.errorMsg = 'Passwords do not match'
+      } else {
+        this.passwordConfirm.errorMsg = '';
+      }
+    },
+    setPasswordConfirm(newValue) {
+      this.passwordConfirm.value = newValue;
+      if(this.passwordConfirm.value != this.password.value) {
+        this.passwordConfirm.errorMsg = 'Passwords do not match'
+      } else {
+        this.passwordConfirm.errorMsg = '';
+      }
+    },
+    uploadAvatar() {
+      this.$refs.fileInput.click();
+    },
+    changeAvatar(event) {
+      const file = event.target.files[0];
+      this.avatar.img = URL.createObjectURL(file);
+    },
+    formValidation() {
+     if(this.username.errorMsg == '' && this.password.errorMsg == '' && this.passwordConfirm.errorMsg == '' && this.email.errorMsg == '') {
+        if(this.username.value.length > 0 && this.password.value.length > 0 && this.passwordConfirm.value.length > 0 && this.email.value.length > 0) {
+          return true;
+        }
+      }
+      return false;
+    },
+    createAccount() {
+      if(this.formValidation()) {
+        //This will be replaced with call to http
+        console.log("Username:"+this.username.value);
+        console.log("Email:"+this.email.value);
+        console.log("Password:"+this.password.value);
+        console.log("ConfirmPassword:"+this.passwordConfirm.value);
+      }
+    }
+  }
+};
+</script>
+
+<style lang="less" scoped>
+@import "../components/common.less";
+
+.register-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80vw;
+
+  .register-page__title {
+    margin: 0.2em 0px;
+    font-size: 2.5em;
+  }
+
+  .register-page__form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    padding: 30px 0px;
+
+    .register-page__input {
+      display: flex;
+      justify-content: center;
+      padding: 0.5em;
+      .input {
+        width: 80%;
+      }
+
+      .register-page__image-upload {
+        display: flex;
+        justify-content: center;
+        align-items: flex-start;
+        flex-direction: column;
+        width: 80%;
+
+        .register-page__image-section {
+          display: flex;
+          justify-content: flex-start;
+          flex-wrap: wrap;
+
+          .register-page__image {
+            width: 200px;
+            height: 200px;
+            margin-right: 30px;
+          }
+        }
+      }
+    }
+  }
+
+  .register-page__bottom {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 50%;
+    padding: 2em 0em;
+
+    .register-page__submit-button {
+      align-self: center;
+    }
+
+    .register-page__link {
+      color: @primary-dark-text-color;
+      font-size: 1.1rem;
+    }
+
+    .invisible-item {
+      display: flex;
+    }
+  }
+}
+
+@media (max-width: 767.98px) {
+
+  .register-page__image-section {
+    align-items: center;
+
+    .register-page__image {
+      order: 2;
+    }
+  }
+}
```

Comment:
Nice! Although right now those guys are plastered together:
![image](https://user-images.githubusercontent.com/24480246/143131723-1f49cce9-01b6-4ca9-9e2b-db06c371c4d1.png)

Introduce a gap and we are good to go?